tools/eos-metrics-collector.exe

```diff
@@ -48,12 +47,12 @@ class OfflineMetrics():
     def metrics_unit(self, operation):
         # TODO: the metrics daemon is also D-Bus activatable, so on stop we
         # should also do something to prevent it being started over D-Bus.
```

Comment:
Yes, but, I believe it's activated via systemd so masking the systemd unit should DTRT.